docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,234 @@
+applications to be successful.
+
+Revenues from coretime sales can be burnt, used to fund the Treasury, split between coretime
+providers (i.e., validators), or used for a mix of those options. The topic is currently under
+discussion. For more information, see [RFC-0010](https://github.com/polkadot-fellows/RFCs/pull/10)
+and [RFC-0015](https://github.com/polkadot-fellows/RFCs/pull/17/files).
+
+## From Chain- to Application-centricity
+
+Polkadot 1.0 was a chain-centric paradigm consisting of isolated chains able to exchange messages.
+This was not fundamentally different from having completely different chains connected to bridges,
+with the only difference of having the relay-chain securing the network, providing message-passing
+capability, and doing some extra tasks such as [crowdloans](../learn/learn-crowdloans.md),
+[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
+[accounts](./learn-accounts-index), [balances](../learn/learn-balance-transfers.md), and
+[governance](../learn/learn-polkadot-opengov.md). Having a chain-centric system will ultimately end
+in chain-centric application and UX.
+
+The true innovation of Polkadot is about leveraging the unique value proposition offered by
+different chains and using those chains’ collaborative potential to build inter-chain applications
+to solve real-world problems. Those applications will thus need to span across chains.
+
+**Increasingly fewer tasks will be handled by the relay-chain** that will focus efforts only on
+primary tasks: securing the network and providing secure message-passing capability.
+[System parachains](../learn/learn-system-chains.md) will be used to take over secondary relay-chain
+tasks such as staking, governance, etc.
+
+### XCM and Accords
+
+[XCM](../learn/learn-xcm-transport.md) is a format, a language of intention abstract over
+functionality common within chains. It creates an expressive language of what you intend to do or
+want to happen. There are no guarantees that chains faithfully interpret XCM messages; thus, XCM is
+not ideal if we are in a trustless environment.
+
+[XCMP](../learn/learn-xcm.md) is the transport layer for delivering XCM messages. It gives the
+transportation method and a secure route but not a framework for binding agreements.
+
+An **Accord** is an _opt-in_ treaty across many chains, where treaty logic cannot be changed or
+undermined by one or more of those chains, and Polkadot guarantees faithful execution of this logic.
+Accords will be specific to a particular function, and any chain that enters the accord will be held
+to it and will service that particular function. To lower the entry barrier, accords can be proposed
+permissionlessly, but because they are opt-in, the accord proposal will take effect until chains
+agree and sign up.
+
+Polkadot is the only ecosystem where accords can properly exist because it has an homogenous
```

Comment:
You have explained XCM, XCMP and Accords above, but there's nothing here that ties all of these 3 concepts together. I was looking for a summary that says something to that effect, but currently there's nothing like that here.

Reply:
added a sentence to tie all these three together